cvat-ui/src/components/labels-editor/label-form.tsx

```diff
@@ -228,20 +264,34 @@ export default class LabelForm extends React.Component<Props> {
         const { key } = fieldInstance;
         const locked = attr.id as number >= 0;
         const existingValues = attr.values;
-
+        const { layout } = this.state;
+        const updatedLayout = layout.map((item, index) => {
+            if (index === key) {
+                return { ...item, h: 2 };
```

Comment:
Do we use two types of `height` to correctly display the validation error?

Lets declare two const variables for the height 1 and 2 with descriptive names